modules/heat_conduction/src/materials/SemiconductorLinearConductivity.C

```diff
@@ -0,0 +1,54 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "SemiconductorLinearConductivity.h"
+#include "libmesh/quadrature.h"
+
+template <>
+InputParameters
+validParams<SemiconductorLinearConductivity>()
+{
+  InputParameters params = validParams<Material>();
+  params.addRequiredCoupledVar("temp", "Variable for temperature.");
+  params.addParam<std::string>("base_name", "Material property base name");
+  params.addRequiredParam<Real>("sh_coeff_A", "Steinhart_Hart coefficient A of the material");
+  params.addRequiredRangeCheckedParam<Real>(
+      "sh_coeff_B",
+      "sh_coeff_B != 0",
+      "Steinhart_Hart coefficient B of the material.  Tt should "
+      "not be zero as it is in denominator.");
```

Comment:
'Tt'->'It'.
I would actually just drop the last sentence about it needing to be nonzero, since the range checking covers that.